README.md

```diff
@@ -71,6 +74,19 @@ pyfuncol provides operations leveraging memoization to improve performance (for
 # {"a": 1, "b": 4, "c": 9}
 ```
 
+### Usage without forbiddenfruit
+
+If you are using a python intepreter other than CPython, forbiddenfruit will not work.
+
+Fortunately, pyfuncol also supports calling the functions directly without extending to builtins.
```

Comment:
Maybe here it would be good to also mention cases in which people do not want to use forbidden fruit for other reason. I added a suggestion